Client/src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <>
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
```

Comment:
Add 'rel="noopener noreferrer"' to anchor tags using 'target="_blank"' to mitigate security risks associated with reverse tabnabbing. [important]